pkg/apis/serving/v1beta1/configmap.go

```diff
@@ -84,6 +85,12 @@ type DeployConfig struct {
 	DefaultDeploymentMode string `json:"defaultDeploymentMode,omitempty"`
 }
 
+type ServiceConfig struct {
+	// ServiceClusterIPNone is a boolean flag to indicate if the service should have a clusterIP set to None.
+	// If the DeploymentMode is Raw, the default value for ServiceClusterIPNone is true when the value is absent.
+	ServiceClusterIPNone *bool `json:"serviceClusterIPNone,omitempty"`
```

Comment:
We probably want another config for the ServiceType, this option only makes sense when type is `ClusterIP`

Reply:
for now, it might be okay to keep this way as we define the service as ClusterIP, with this flag the only thing we do is set it to None.

Or, what do you have in mind? Open the configuration for more properties?

Reply:
Good idea.